pkg/management/server.go

```diff
@@ -144,7 +144,9 @@ func NewServer(
 		grpc.Creds(insecure.NewCredentials()),
 		grpc.UnknownServiceHandler(unknownServiceHandler(director)),
 		grpc.ChainStreamInterceptor(otelgrpc.StreamServerInterceptor()),
-		grpc.ChainUnaryInterceptor(otelgrpc.UnaryServerInterceptor()),
+		grpc.ChainUnaryInterceptor(
+			util.NewClientGrpcTtlCacher(nil).UnaryServerInterceptor(),
```

Comment:
Is this arg supposed to be nil?

Reply:
yeah; we bundle both interceptors into one interface. Since the caching implementation is entirely client-side in these interceptors, when we only require a ServerInterceptor declaration we can omit the caching implementation.

Could add comment or implement a no-op cache to make this clearer.